docs/source/changelog.md

```diff
@@ -1,7 +1,35 @@
-# ipywidgets changelog
+# ipywidgets Changelog
 
 A summary of changes in ipywidgets. For more detailed information, see the issues and pull requests for the appropriate milestone on [GitHub](https://github.com/jupyter-widgets/ipywidgets).
 
+## 7.6
```

Comment:
I think this changelog entry comes in with #3086 too, so hopefully there is no conflict.

Reply:
Do you want to land #3086 now and I can rebase or do you want me to merge this now.

Reply:
I was giving sylvain or vidar time for one more day to look at it before merging.

Reply:
Pretty sure this is just a complete copy/paste. How adventurous do you feel?

Reply:
Except for the title.

Reply:
I'll land this after #3086.